app/controllers/exchanges/hbx_profiles_controller.rb

```diff
@@ -556,9 +556,12 @@ def mark_pvp_eligibilities
               else
                 { success: l10n('hbx_profiles.mark_pvp_success') }
               end
+    org = BenefitSponsors::Organizations::ExemptOrganization.by_employer_profile(@product.issuer_profile.id).last
```

Comment:
You might consider renaming this method in Organization, as it's not actually implemented as specific to 'employer profiles' and as you've done here, it's usable for other profile types. I'd perhaps rename to simply "by_profile". (or alias to an issue-specific version), to avoid confusion.